core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -444,6 +444,10 @@ public JournalEntry next() {
         if (!hasNext()) {
           throw new NoSuchElementException();
         }
+        /*
+         * When the BlockStore is RocksBlockMetaStore, the iterator takes care of thread safety
+         * in hasNext() and next() calls.
+         */
```

Comment:
i will add comment like this on 2 places:
1. RocksUtils where the iter is created
2. Where this iter is used